packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -57,6 +57,10 @@ const Dropdown = forwardRef(
 								<DropdownButton
 									{...entryRest}
 									{...menu}
+									onClick={event => {
+										menu.hide();
+										entry?.onClick(event);
```

Comment:
`entry` can't be null/undefined  or it breaks before.

```suggestion
										entry.onClick(event);
```

Reply:
Fair point